graph/mutators/comment.js

```diff
@@ -169,16 +169,26 @@ const createPublicComment = (context, commentInput) => {
  * @param {String} status      the new status of the comment
  */
 
-const setCommentStatus = ({comment}, {id, status}) => {
-  return CommentsService.setStatus(id, status)
-    .then(res => res);
+const setCommentStatus = ({loaders: {Comments}}, {id, status}) => {
+  return CommentsService
+    .setStatus(id, status)
+    .then((comment) => {
+
+      // If the loaders are present, clear the caches for these values because we
```

Comment:
This is a good pattern so long as we have a relatively small number of places to clear these caches. As the number of cache clearing triggers grows, let's consider refactoring this cache layer into a service that exposes a tight api.